Package.swift

```diff
@@ -17,7 +17,8 @@ let package = Package(
   name: "swift-format",
   products: [
     .executable(name: "swift-format", targets: ["swift-format"]),
-    .library(name: "SwiftFormat", targets: ["SwiftFormat"]),
+    .library(name: "SwiftFormat", targets: ["SwiftFormat", "SwiftFormatConfiguration"]),
+    .library(name: "SwiftFormatConfiguration", targets: ["SwiftFormatConfiguration"]),
```

Comment:
Does it make sense to ever want `SwiftFormatConfiguration` without `SwiftFormat`?

Reply:
My use case for that is a product that only needs to be able to dynamically assemble a configuration and convert it to JSON for export. The larger SwiftFormat library and its dependencies on SwiftSyntax and SwiftPM are massive overkill for this; they take a long time to fetch and build.